design-documents/asynchronous-import/base-extension.md

```diff
@@ -41,71 +44,38 @@ Path is relative from Magento Root folder
 
 ```
 {
-  "source": {
-      "import_data": "http://some.domain/file.csv",
-      "import_type": "external",
-      "uuid": "UUID",
-      "format": {
-          "csv_separator": "string",
-          "csv_enclosure": "string",
-          "csv_delimiter": "string",
-          "multiple_value_separator": "string"
-      }
+  "uuid": "UUID",
+  "source_data": {
+    "source_type": "remote",
+    "source_data": "http://some.domain/file.csv" 
+  } 
+  "format": {
+    "delimiter": "string",
+    "enclosure": "string",
+    "escape": "string",
+    "multiple_value_separator": "string"
   }
 }
 ```
 
-#### Base64 encoded file content 
+#### Upload file (Base64 encoded)
 
 ```
 {
-  "source": {
-      "import_data": "c2t1LHN0b3JlX3ZpZXdfY29kZSxhdHRyaWJ1dGVfc2V0X2NvZGUscHJvZHVjdF90eXBlLGNhdGVnb3JpZXMscHJvZHVjdF93ZWJzaXRlcyxuYW1lLGRlc2NyaXB0aW9uLHNob3J0X2Rlc2NyaXB0aW9uLHdlaWdodCxwcm9kdWN0X29ubGluZSx0YXhfY2xhc3NfbmFtZSx2aXNpYmlsaXR5LHBya......",
-      "import_type": "base64_encoded_data",
-      "uuid": "UUID",
-      "format": {
-          "csv_separator": "string",
-          "csv_enclosure": "string",
-          "csv_delimiter": "string",
-          "multiple_value_separator": "string"
-      }
+  "uuid": "UUID",
+  "source_data": {
+    "source_type": "upload_file",
+    "source_data": "c2t1LHN0b3JlX3ZpZXdfY29kZSxhdHRya..." 
+  } 
+  "format": {
+    "delimiter": "string",
+    "enclosure": "string",
+    "escape": "string",
+    "multiple_value_separator": "string"
```

Comment:
what is the purpose of this "multiple_value_separator"?

Reply:
For example Multiselect attribute value: 
"color": "Red|Green|White"
In this case - | is a multiple value separator

Reply:
okay, so it should be multiple_value_delimiter? to match above convention

Reply:
Make sense, aligned